src/test/java/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApprovalTest.java

```diff
@@ -136,6 +143,22 @@ public void malformedScriptApproval() throws Exception {
         assertEquals(0, sa.getDangerousApprovedSignatures().length);
     }
 
+    @Issue("JENKINS-57563")
+    @LocalData("upgrade")
+    @Test
+    public void upgradeSmokes() throws Exception {
+        r.jenkins.setSecurityRealm(r.createDummySecurityRealm());
+        r.jenkins.setAuthorizationStrategy(new MockAuthorizationStrategy().
```

Comment:
I don't think Jenkins' security realm or authorization strategy matter for this test since we are running in the sandbox, so these lines could just be deleted.